aksel.nav.no/website/pages/eksempler/primitive-bleed/directions.tsx

```diff
@@ -47,15 +47,13 @@ function DemoWrapper({ children }: { children: React.ReactNode }) {
 }
 
 // EXAMPLES DO NOT INCLUDE CONTENT BELOW THIS LINE
-export default withDsExample(Example, {
-  showBreakpoints: true,
```

Comment:
I think we should only show the breakpoints in examples where breakpoints matters.